### ISSUE TITLE
default value of 5.x installs nodejs 0.10

### ISSUE THREAD

**Issue by jrwren** [battlemidget/juju-layer-node#1]
using: `includes: ['layer:nodejs']` in layer.yaml

the resulting charm deploys and installs nodejs version 0.10.x

```
$ juju ssh 1 dpkg -p nodejs | grep Version
Version: 0.10.25~dfsg2-2ubuntu1
```

But juju get shows the value is 5.x:

```
$ juju get example-charm
settings:
  app-path:
    default: true
    description: |
      The default location of your Node.js application, defaults /srv/app
    type: string
    value: /srv/app
  node-version:
    default: true
    description: |
      NodeJS Version to use for your application. Supported versions are
      0.10, 0.12, 4.x, 5.x
    type: string
    value: 5.x

```

**Comment by adam-stokes** [comment]
Thanks will take a look

**Comment by adam-stokes** [comment]
This would be fixed by #4

**Comment by adam-stokes** [comment]
Fixed with latest commit
